Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.cpp

```diff
@@ -39,23 +39,56 @@ namespace AZ
-            m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Ll][Oo][Dd]3(_optimized)?$", PatternMatcher::MatchApproach::Regex, "LODMesh3", true));
-            m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Ll][Oo][Dd]4(_optimized)?$", PatternMatcher::MatchApproach::Regex, "LODMesh4", true));
-            m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Ll][Oo][Dd]5(_optimized)?$", PatternMatcher::MatchApproach::Regex, "LODMesh5", true));
-            m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Pp][Hh][Yy][Ss](_optimized)?$", PatternMatcher::MatchApproach::Regex, "PhysicsMesh", true));
-            m_softNames.push_back(aznew NodeSoftNameSetting("_ignore", PatternMatcher::MatchApproach::PostFix, "Ignore", false));
-            // If the filename ends with "_anim" this will mark all nodes as "Ignore" unless they're derived from IAnimationData. This will
-            // cause only animations to be exported from the source scene file even if there's other data available.
-            m_softNames.push_back(aznew FileSoftNameSetting("_anim", PatternMatcher::MatchApproach::PostFix, "Ignore", false,
-                { FileSoftNameSetting::GraphType(SceneAPI::DataTypes::IAnimationData::TYPEINFO_Name()) }));
 
+            PopulateDefaultSoftNameSettings();
             m_UseCustomNormals = true;
         }
 
+        void SceneProcessingConfigSystemComponent::PopulateDefaultSoftNameSettings()
+        {
+            auto settingsRegistry = AZ::SettingsRegistry::Get();
+            if (settingsRegistry)
+            {
+                AZStd::vector<NodeSoftNameSetting*> nodeSoftNameSettings;
+                settingsRegistry->GetObject(nodeSoftNameSettings, AssetProcessorDefaultNodeSoftNameSettingsKey);
+                for (NodeSoftNameSetting* nodeSoftNameSetting : nodeSoftNameSettings)
+                {
+                    if (!AddSoftName(nodeSoftNameSetting))
+                    {
+                        delete nodeSoftNameSetting;
+                        nodeSoftNameSetting = nullptr;
+                    }
+                }
+
+                AZStd::vector<FileSoftNameSetting*> fileSoftNameSettings;
+                settingsRegistry->GetObject(fileSoftNameSettings, AssetProcessorDefaultFileSoftNameSettingsKey);
+                for (FileSoftNameSetting* fileSoftNameSetting : fileSoftNameSettings)
+                {
+                    if (!AddSoftName(fileSoftNameSetting))
+                    {
+                        delete fileSoftNameSetting;
+                        fileSoftNameSetting = nullptr;
+                    }
+                }
+            }
+
+            if (m_softNames.empty())
+            {
+                // Defaults in case there's no config setup
+                m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Ll][Oo][Dd]_?1(_optimized)?$", SceneAPI::SceneCore::PatternMatcher::MatchApproach::Regex, "LODMesh1", true));
```

Comment:
I'd like to see these defaults moved out of C++ into a setreg or other data file, to let technical artists manage the defaults, too. If that's too much to do as part of this change, then I can cut a ticket for the backlog.

Reply:
The setreg file that included in this PR already include all the default soft name settings, technical artists can just update this file to manage the soft name settings they want to use. These default values in code will only be used when technical artists delete or empty this default setreg file, so I'm not sure about the reason to put the default list in another setreg file.

Reply:
I would posit that sensible defaults values be set in the setting could not be found in the Registry.

The [SettingsRegistryInterface::GetObject](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Settings/SettingsRegistry.h#L274-L281) function returns a boolean to indicate whether the setting has been successfully read from the registry.

What could be done is to check that if read has failed, then initialize the defaults in that case

Reply:
Based on my last comment, I would say a complete working set of default values are needed, but instead a set of default values that doesn't result in uninitialized data being used or data that would cause a crash.